### DISCUSSION TITLE
audio max_duration_sec

### DISCUSSION THREAD

**Discussion by xiabo2011** [serp-ai/bark-with-voice-clone#79] [Q&A]
Hi, I'm wondering why the audio max_duration_sec is 15.12s. Could you help with some details?

**Comment by gustrd** [comment]
I believe that's about the model's context window, like ChatGPT has it's prompt size limit.

**Reply by xiabo2011** [reply]
Thanks for replying, do you know exactly how this result is calculated? I mean 15.12 sec, is precise to two decimal points, not 15 sec or 16 sec.

**Reply by gustrd** [reply]
I don't know. Maybe at Bark's main repo they can answer.

**Comment by devinschumacher** [comment]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik

@gustrd @xiabo2011
